LetsDoStuff.Domain/Models/Activity.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+
+namespace LetsDoStuff.Domain.Models
+{
+    public class Activity 
+    {
+        public int Id { get; set; }
+
+        /// <summary>
+        /// Gets or sets Name of the activity.
+        /// </summary>
+        [Required]
+        public string Name { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Description of the activity.
+        /// </summary>
+        [Required]
+        public string Description { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Capacity - number of people taking part in the activity.
+        /// </summary>
+        [Required]
+        public int Capacity { get; set; }
+
+        /// <summary>
+        /// Gets or sets the Creator.
+        /// </summary>
+        public User Creator { get; set; } 
+
+        /// <summary>
+        /// Gets or sets the list of ActivityTags.
+        /// </summary>
+        public List<ActivityTag> ActivityTags { get; set; }
+
+        /// <summary>
+        /// Gets or sets the list of Tags.
+        /// </summary>
+        public List<Tag> Tags { get; set; }
```

Comment:
Are you sure you need both of the two lists here?

Reply:
> Are you sure you need both of the two lists here?

https://daveabrock.com/2020/08/29/dotnet-stacks-14

Reply:
Do we use net5 as in this article you gave?

Reply:
> Do we use net5 as in this article you gave?

> In the standup, Arthur Vickers mentioned an important fact that’s easy to overlook: while the release timing is similar, EF Core 5 is not bound to .NET 5. It targets .NET Standard 2.1, so can be used in any .NET Core 3.x apps (and of course is available for use in .NET 5).

From the article.